cli/test/smokehouse/test-definitions/hsts-fully-present.js

```diff
@@ -0,0 +1,26 @@
+/**
+ * @license
+ * Copyright 2024 Google LLC
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/**
+ * @type {Smokehouse.ExpectedRunnerResult}
+ * Expected Lighthouse results a site with full HSTS deployed.
+ */
+const expectations = {
+  lhr: {
+    requestedUrl: 'https://hstspreload.org/',
```

Comment:
Should we be worried about the HSTS policy changing on this site?

Not a huge deal, but it could cause the tests to start failing if something changes on this site.

Reply:
This one should be fine actually. It's managed by Google for the preload lists, so I wouldn't expect it to regress for HSTS.